src/util/googleSpreadsheetLayer.js

```diff
@@ -102,10 +102,16 @@ L.SpreadsheetLayer = L.LayerGroup.extend({
       var $ = window.jQuery;
       var ssURL = self.options.url || '';
       self._map.spin(true);
+      var timeout = setTimeout(function() {
```

Comment:
@sagarpreet-chadha, what do you think about adding the changes to this util layer?